Dockerfile

```diff
@@ -15,4 +19,6 @@ RUN dotnet publish -r linux-musl-x64 -c Release -o /opt/test-runner
 FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine AS runtime
 WORKDIR /opt/test-runner
 COPY --from=build /opt/test-runner/ .
+COPY --from=build /usr/local/bin/ /usr/local/bin/
```

Comment:
Probably a little wide, but I'm guessing so far our little web server is the only thing in /user/local/bin? :-)

Reply:
Yep :)

Reply:
Wait does this need the `EXEC true` hack in-between? @iHiD

Reply:
Apparently not (as it built) 🤷